.github/workflows/queue.yml

```
@@ -14,846 +14,9 @@ permissions:
-            -action update_state
-            -ref "${{ needs.context.outputs.head_sha }}"
-            -owner "${{ needs.context.outputs.base_repo_owner }}"
-            -repo "${{ needs.context.outputs.base_repo_name }}"
-            -state ${{ steps.status.outputs.result }}
-            -context "test/terratest"
-            -description "Test run finished"
-            -url "https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-
+    uses: ./.github/workflows/terratest.yml
```

actionlint (1.7.4)

could not read reusable workflow file for "./.github/workflows/terratest.yml": open /home/jailuser/git/.github/workflows/terratest.yml: no such file or directory

(workflow-call)